docs/v0.7/en/contributor-guide/datanode/metric-engine.md

```diff
@@ -0,0 +1,39 @@
+# Metric Engine
+
+## Overview
+
+The `Metric` engine is a component of GreptimeDB, and it's an implementation of the "storage engine". It mainly targets scenarios with a large number of small tables for observable metrics.
```

Comment:
```suggestion
The `Metric` engine is a component of GreptimeDB, and it's an implementation of the _storage engine_. It mainly targets scenarios with a large number of small tables for observable metrics.
```

And others.

Or @nicecui has different docs guidelines >_<

Reply:
Just the `storage engine` is ok. we do not need `"storage engine"` or `_storage engine_`.
If the product name can be translated into other languages, then use the name in plain format without emphasis.
This will save the author most of the energy to focus on the content logic of the feature.